lib/ansible/plugins/filter/mathstuff.py

```diff
@@ -123,14 +129,28 @@ def union(environment, a, b):
     return c
 
 
-def min(a):
-    _min = __builtins__.get('min')
-    return _min(a)
+@environmentfilter
+def min(environment, a, case_sensitive=False, attribute=None):
```

Comment:
at this point i would use kwargs to avoid 'suprises' from the jinja2 verision, also easier to 'error' if any kwargs supplied but we dont have new enough jinja2

Reply:
Works for me, should simplify things a bit. If we're keeping it generic, should the current args like `attribute` and `case_sensitive` still be tested and documented?